play/homepage_template.html

```diff
@@ -46,6 +46,7 @@ <h3>{{subtitle_2}}</h3>
     <li>{{feature_05}}</li>
     <li>{{feature_06}}</li>
     <li>{{feature_07}}</li>
+    <li>{{feature_13}}</li>
     <li>{{feature_08}}</li>
```

Comment:
Will need to remove this? Looks like it's removed (merged) in `playstore-description.yml`